### ISSUE TITLE
[ModernSearch] property pane textboxes lose focus per character

### ISSUE THREAD

**Issue by wobba** [SharePoint/sp-dev-solutions#96]
#### Category
- [ ] Question
- [x] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
I expect to be able to type as much as I want in a text box without focus being lost on r-etrigger of search.

#### Observed Behavior
![image](https://user-images.githubusercontent.com/51104/55079272-20fbbd80-509c-11e9-8587-b7728924df58.png)
Each key stroke when specifying for example manage properties remove focus from the input box.

@FranckyC or @YannickRe, any ideas on this?

**Comment by FranckyC** [comment]
@wobba must be caused by a focus on the search result WP after a new query. Linked to #95 and componentWillReceiveProps I think.

**Comment by YannickRe** [comment]
I assume it's because of the last pr by @FranckyC .  My code had:
```
                    if (isPageChanged)
                    {
                        // Set the focus at the top of the component
                        this._searchWpRef.focus();
                    }
```
but it is changed to remove the if statement. Also, the property pane is reactive.
As a result, each change to a property pane field triggers the search and puts the focus on top of the search results webpart.

Probably need to correctly fix #95 or make the property pane non-reactive.
